src/models/healthCert.ts

```diff
@@ -25,11 +25,18 @@ export const getParticularsFromHealthCert = (document: HealthCertDocument) => {
       identifierEntry.type instanceof Object &&
       identifierEntry.type.text === "NRIC"
   );
+
+  const fin = patient?.identifier?.find(
+    identifierEntry =>
+      identifierEntry.type instanceof Object &&
+      identifierEntry.type.text === "FIN"
+  );
+
   const ppn = patient?.identifier?.find(
     identifierEntry => identifierEntry.type === "PPN"
   );
 
-  return { nric: nric?.value, passportNumber: ppn?.value };
+  return { nricOrFin: nric?.value || fin?.value, passportNumber: ppn?.value };
```

Comment:
Nit: my only concern about using a single field to store both possible values is that the caller won't be able to distinguish nric or fins.

I don't know if that'll ever be a use case for it. If there is, I was thinking of having separate fields for `nric` or `fin`, and using whichever one is defined as the input to the notification function.

Reply:
Hmm that's a good point. Let me separate the `nric` and `fin` field so it is clearer down the road.